modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaJAXRSSpecServerCodegen.java

```diff
@@ -51,6 +52,7 @@
     private boolean interfaceOnly = false;
     private boolean returnResponse = false;
     private boolean generatePom = true;
+    private boolean generateBuilders = true;
```

Comment:
If you default this setting to false, you can then target this change to the current master as it's no longer a breaking change. Otherwise, you will need to wait for 4.3.0 to be released on Feb 29.